### ISSUE TITLE
Hardcore Ender Dragon breaks OpenBlocks graves.

### ISSUE THREAD

**Issue by border999** [chylex/Hardcore-Ender-Expansion#130]
During my fight with the enhanced dragon fight, I was killed, the next time I went in was to attempt to kill the dragon again, skipping to the problem. I have death waypoints, I went to where the grave should have been, I found a few items. That's it, the graves all broke.
FIX THIS, THIS IS A BIG PROBLEM. THESE GRAVES WERE CODED TO SURVIVE THE WITHER. I recommend you patch this fast.

**Comment by chylex** [comment]
Not a priority for a quickfix, probably not going to be updated until the redesign or if a major crash is found in current version.

**Comment by border999** [comment]
But it will be fixed in the next update correct?

**Comment by chylex** [comment]
It will be possibly fixed once the redesign is finished, which can take a month or two. Not planning any updates for the older versions unless they're critical crashes, in that case I would also include this fix.

**Comment by chylex** [comment]
I'm including the fix in next bugfix update. Keep in mind that if a mod is checking specifically for the dragon, my dragon is different so it will break the block anyways, but things like graves should be protected from anything in which case they'll be fine.

**Comment by border999** [comment]
thank you for your time

**Comment by chylex** [comment]
Actually, the graves might not have been spawning at all. Are you sure the graves were there and broken by the dragon?

**Comment by border999** [comment]
There's no actual way of making sure that they were deployed, if the dragon basically destroys it the second the player dies. What makes you question if the graves deployed?
